src/Interpreters/QueryLog.cpp

```diff
@@ -59,41 +59,64 @@ ColumnsDescription QueryLogElement::getColumnsDescription()
     auto low_cardinality_string = std::make_shared<DataTypeLowCardinality>(std::make_shared<DataTypeString>());
     auto array_low_cardinality_string = std::make_shared<DataTypeArray>(low_cardinality_string);
 
-    return ColumnsDescription
-    {
+    return ColumnsDescription{
         {"hostname", low_cardinality_string, "Hostname of the server executing the query."},
         {"type", std::move(query_status_datatype), "Type of an event that occurred when executing the query."},
         {"event_date", std::make_shared<DataTypeDate>(), "Query starting date."},
         {"event_time", std::make_shared<DataTypeDateTime>(), "Query starting time."},
         {"event_time_microseconds", std::make_shared<DataTypeDateTime64>(6), "Query starting time with microseconds precision."},
         {"query_start_time", std::make_shared<DataTypeDateTime>(), "Start time of query execution."},
-        {"query_start_time_microseconds", std::make_shared<DataTypeDateTime64>(6), "Start time of query execution with microsecond precision."},
+        {"query_start_time_microseconds",
```

Comment:
I am absolutely not against code re-formatting but if it is a big one, then we rather do it in a separate PR.